.github/workflows/image.yml

```diff
@@ -75,3 +76,35 @@ jobs:
             --tag ghcr.io/getsentry/conduit-${{ matrix.service }}:dev \
             ghcr.io/getsentry/conduit-${{ matrix.service }}:${{ github.event.pull_request.head.sha || github.sha }}-amd64-dev \
             ghcr.io/getsentry/conduit-${{ matrix.service }}:${{ github.event.pull_request.head.sha || github.sha }}-arm64-dev
+
+# Copy manifest from GHCR to Google Artifact Registry, referenced Relay (https://github.com/getsentry/relay/blob/master/.github/workflows/ci.yml)
+  copy-manifest-to-gar:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        service: [gateway, publish]
+    needs: [assemble]
+    if: ${{ (github.ref_name == 'main' || startsWith(github.ref_name, 'releases/')) && github.event_name != 'pull_request' }}
+    permissions:
+      contents: read
+      id-token: write
+    steps:
+      - name: Google Auth
+        uses: google-github-actions/auth@v3
```

Comment:
Let's pin these to specific shas of the latest versions similar to how other workflows in the repo do it